index.html

```diff
@@ -48,6 +48,109 @@ <h1 class="logo-text">Pascal</h1>
         </div>
     </section>
 
+    <section class="work">
+
+        <div class="card">
+            <div class="card-header">
+            </div>
+            <h2 class="card-titles">TONIC</h2>
+
+
+            <ul class="bandle">
+                <li>
+                    <p class="canopy">CANOPY</p>
+                </li>
+                <li><img src="./img/cannopy.png" class="canopy-circle" alt="canopy" /></li>
+                <li>
+                    <p class="role">Backend Dev</p>
+                </li>
+                <li><img src="./img/cannopy.png" class="canopy-circle" alt="canopy" /></li>
+                <li>
+                    <p class="role">2015</p>
+                </li>
+            </ul>
+
+            <div class="card-content">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="content_btn">
+                <li class="data for-html">
+                    <p>html</p>
+                </li>
+                <li class="data for-css">
+                    <p>CSS</p>
+                </li>
+                <li class="data for-js">
+                    <p>JavaSript</p>
+                </li>
+            </ul>
+            <div class="see-project">
+                <a href="#" class="see">See Project</a>
+            </div>
```

Comment:
- [ ] Please make sure to fix what my predecessor requested previously. 
- [ ] Original Design:
![d1](https://user-images.githubusercontent.com/63972670/159092742-c3afcf1e-d3e3-4912-a91e-3a568dc3be6c.JPG)
- [ ]  Your design:  
![d2](https://user-images.githubusercontent.com/63972670/159092772-1b68b9d3-5b2c-49a1-86b5-cfcf37cf07bb.JPG)